apps/mainsite/settings.py

```diff
@@ -277,59 +276,70 @@ def legacy_boolean_parsing(env_key, default_value):
 
 LOG_STORAGE_DURATION = 30  # days
 
+handlers = {
+    'badgr_events': {
+        'level': 'INFO',
+        'formatter': 'json',
+        'class': 'logging.handlers.TimedRotatingFileHandler',
+        'when': 'H',
+        'interval': 1,
+        'backupCount': 30 * 24,  # 30 days times 24 hours
+        'filename': os.path.join(LOGS_DIR, 'badgr_events.log'),
+    },
+    'badgr_debug': {
+        'level': 'DEBUG',
+        'formatter': 'badgr',
+        'class': 'logging.handlers.TimedRotatingFileHandler',
+        'when': 'H',
+        'interval': 1,
+        'backupCount': 30 * 24,  # 30 days times 24 hours
+        'filename': os.path.join(LOGS_DIR, 'badgr_debug.log'),
+    },
+    'badgr_debug_console': {
+        'level': 'DEBUG',
+        'formatter': 'default',
+        'filters': ['require_debug_true'],
+        'class': 'logging.StreamHandler',
+    },
+}
+
+debug_handlers = ['badgr_debug']
+SERVER_NAME = os.environ['SERVER_NAME']
+LOKI_URL = 'http://195.169.124.131:3100/loki/api/v1/push'
+
+if DOMAIN.startswith('acc') or DOMAIN.startswith('prod'):
+    handlers = handlers | {
```

Comment:
Would it not make more sense to:

* first set up a few handlers objects (console_handler, badgr_events_handler etc)
* then set up loggers using one or more of these handlers
* then use that list to set up the actual LOGGING for this env (prod, dev acc etc) using simple conditionals (prod, acc etc.

That would make it even easier to and cleaner than this PR already makes it, IMHO?

Reply:
Will look into it

Reply:
Agreed to look into it again at some later point.